src/routes/reference/components/portal.mdx

```diff
@@ -17,6 +17,8 @@ function Portal(props: {
 
 This inserts the element in the mount node. Useful for inserting Modals outside of the page layout. Events still propagate through the Component Hierarchy.
 
+`<Portal>` will only run on the client and has hydration disabled.
+
 The portal is mounted in a `<div>` unless the target is the document head. `useShadow` places the element in a Shadow Root for style isolation, and `isSVG` is required if inserting into an SVG element so that the `<div>` is not inserted.
```

Comment:
```suggestion
This inserts the element in the mount node. 
Useful for inserting Modals outside of the page layout. 
Events still propagate through the component hierarchy, however `<Portal>` will only run on the client and has hydration _disabled_.

The portal is mounted in a `<div>` unless the target is the document head. 
`useShadow` places the element in a Shadow Root for style isolation, and `isSVG` is required if inserting into an SVG element so that the `<div>` is not inserted.
```